ci/jenkins/test-vmc.sh

```diff
@@ -466,26 +466,6 @@ function deliver_antrea {
     done
 }
 
-function run_integration {
```

Comment:
Jenkins integration test is recovered and I guess we can keep it for manual testing on jenkins?

Reply:
The point of making this change is to reduce the maintanence efforts on jenkins integration testbeds, otherwise it would just make testing more complicated.
I think there is no need to maintain two kinds of runners if Github Action can meet the requirement, as there would be no reason to manual run it on jenkins (Github Action supports re-running failed test).

Reply:
I see, we can remove jenkins manual integration job if it supports re-running on Github Action, thanks.